### ISSUE TITLE
Cannot trigger VoiceOver (Cmd-[Fn-]F5) with laptop keyboard

### ISSUE THREAD

**Issue by lgarron** [tekezo/Karabiner-Elements#396]
The command for VoiceOver is Cmd-F5.

My [external keyboard](https://www.amazon.com/Microsoft-L5V-00001-Ergonomic-Wireless-Keyboard/dp/B00CYX54C0/ref=sr_1_1/155-2826007-3549805?ie=UTF8&qid=1478200564&sr=8-1&keywords=sculpt+keyboard) can trigger this when Karabiner-Elements is not running.

However, when Karabiner-Elements is running, this does *not* work.
Whether I set F5 in the Karabiner-Elements "Function Keys" pref pane to "vk_consumer_illumination_down" or "F5", it does not work anymore.

Is there a way to avoid doing any function key mapping at all?

**Comment by lgarron** [comment]
It seems the next filed issue (#397) also had a similar issue.
I had no problems before Karabiner-Elements tried to modify the function keys; would it be possible to add an option to prevent any function key remapping?

**Comment by ushiushix** [comment]
In my environment, command+F5 is toggles VoiceOver using external keyboard.
I configured function keys to work as F1 to F12 in the system preferences.

**Comment by lgarron** [comment]
I can trigger VoiceOver using Cmd-F5 on my external keyboard with "Use all F1, F2, etc. keys as standard function keys" checked.

However, it breaks Cmd-[fn-]F5 on the laptop keyboard itself – i.e. Cmd-[fn-]F5 triggers VoiceOver when Karabiner is *not* running, but doesn't work when Karabiner is running.

**Comment by lgarron** [comment]
This is still broken on the laptop keyboard. 😞

**Comment by lgarron** [comment]
Still an issue!

**Comment by lgarron** [comment]
Still an issue, I believe.

**Comment by lgarron** [comment]
Also still an issue!
